favorite_files.py

```diff
@@ -365,24 +388,32 @@ def run(self):
 class TogglePerProjectFavoritesCommand(sublime_plugin.WindowCommand):
     """Toggle per project favorites."""
 
-    def run(self):
+    def run(self, restarted=False):
         """Run the command."""
         win_id = self.window.id()
 
         # Try and toggle back to global first
-        if not Favs.toggle_global(win_id):
+        if not restarted and not Favs.toggle_global(win_id):
```

Comment:
This looks like its related to project toggling which is out of scope for this pull.  Also, FavFileMgr was not meant to be called directly from this layer.